### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by franclobo**
### To-Do List: Structure
Set up a new project with webpack that is based on the web pack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To-Do List placeholder (ul element). The index.html file must be set as an HTML Webpack Plugin template.
Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On the page, load renders the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To-Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash';
+import "./style.css";
+const formList = document.querySelector(".form");
+const addNew = document.querySelector(".input");
+const container = document.querySelector(".list");
+const clearAll = document.querySelector(".clear");
+const upDate = document.querySelector(".refresh");
+
+let list = JSON.parse(localStorage.getItem("list"));
```
Comment (zainsadaqat) [inline src/index.js:9]: Todo items are not being stored in the local storage and hence the items are not being displayed. It's just a static page on my side.
#### As you can see in the screenshot
Screen Shot 2022-05-08 at 8 12 27 AM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌
# Hello 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that all linters are not passing and are not green. Kindly fix this.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello  @franclobo 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Todo list items are not being displayed nor stored in the local storage. Kindly fix this.
Screen Shot 2022-05-08 at 9 45 37 AM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
```
Comment (keroloussamy) [inline src/index.js:4]: - [x] Kindly for this milestone you should have an array of simple data on your index.js and a function that display this data. Kindly add them as asked by the requirements.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @franclobo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (franclobo) [reply src/index.js:4]: Done

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @franclobo 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working on the required changes from the previous reviews

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**